truss/templates/shared/secrets_resolver.py

```diff
@@ -58,3 +56,12 @@ def __iter__(self):
 
     def __len__(self):
         return len(self._base_secrets)
+
+
+def _secret_missing_error_message(key: str) -> str:
+    return f"""
+Secret '{key}' not found. Please check that:
+  * The secret is defined in the secrets section of your config file
+  * The model was pushed with the --trusted flag
```

Comment:
Should we also add that `* Secret `{key}` is defined in the secret manager`?

Reply:
yeah i like that